macosx/InfoActivityViewController.mm

```diff
@@ -180,50 +179,68 @@ - (CGFloat)vertLayoutHeight
     return NSHeight(self.fTransferView.frame) + NSHeight(self.fDatesView.frame) + (2 * kStackViewInset) + kStackViewVerticalSpacing;
 }
 
-- (CGFloat)changeInWindowHeight
+- (void)updateOldHeight:(CGFloat)height
+{
+    self.oldHeight = height;
+}
```

Comment:
Wouldn't it be `setOldHeight:` and `_oldHeight = height`?

Reply:
Actually, removing this method entirely and just moving `@property(nonatomic) CGFloat oldHeight;` to the header should be good.

Reply:
no problem, Ive also updated all the other properties to fit better with the project conventions